src/OrchardCore.Modules/OrchardCore.Sitemaps/Views/CustomPathSitemapSource.Edit.cshtml

```diff
@@ -0,0 +1,53 @@
+@model CustomPathSitemapSourceViewModel
+@using OrchardCore.Sitemaps.Models
+<h5>@T["Custom Url"]</h5>
+
+@if (String.IsNullOrEmpty(Site.BaseUrl))
+{
+    <p class="alert alert-warning">@T["The site settings 'Base url' property is empty, but is required to generate sitemaps."]</p>
+}
+
+<div id="index-all-row" class="row">
+    <div class="col-6">
+        <div class="card mb-4">
+            <div class="card-body">
+                <div class="form-group" asp-validation-class-for="Url">
+                    <label asp-for="Url">@T["Url"] </label>
+                    <div class="input-group">
+                        @if (!String.IsNullOrWhiteSpace(Site.BaseUrl))
+                        {
+                            <div class="input-group-prepend">
+                                <div class="input-group-text">@Site.BaseUrl</div>
+                            </div>
+                        }
+                        <input asp-for="Url" class="form-control" autofocus />
+                        <span asp-validation-for="Url" class="text-danger">@T["The Url is required or have used a wrong format"]</span>
```

Comment:
To make it easy on the translators try and get the same text as is used to validate the sitemap.path